app/lib/features/settings/pages/email_addresses.dart

```diff
@@ -63,6 +63,7 @@ class _AddEmailAddrState extends State<AddEmailAddr> {
 
   void onSubmit(BuildContext context) {
     if (!_formKey.currentState!.validate()) {
+      _log.severe('Email or password seems to be not valid.');
```

Comment:
Not log worthy

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2068/commits/ba0fb79ec4c7b852142c49b9af2b9645a33a0196